app/views/user/stores/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<% @stores.each do |store| %>
+  <%= store.name %> (<%= store.status %>)<br>
```

Comment:
This reminded me. Did we ever ask @corywest if he wants the store to have any attributes besides name, url and status?

Reply:
@anlewi5 you're right, we should definitely ask. Though also, I think that his response won't necessarily require a change for this ticket